docs/pipelineruns.md

```diff
@@ -678,6 +680,7 @@ Conditions:
   Type:                  Succeeded
 Skipped Tasks:
   Name:       skip-this-task
+  Reason:     WhenExpressionsSkip
```

Comment:
Does the user see "WhenExpressionsSkip" or do they see "When Expressions evaluated to false"? (looks to me like the latter but want to double check). If the latter, would you mind updating this comment? If the former, the full sentence might be more informative to show.

Reply:
Done! Thanks for pointing that out.